src/util/setup.ts

```diff
@@ -1,31 +1,25 @@
 // describe the shape of libraries not written in TypeScript
-declare
-var emit:any;
-declare
-var getRow:any;
-declare
-var send:any;
-declare
-var sum:any;
-declare
-var toJSON:any;
+declare var emit:any;
+declare var getRow:any;
+declare var send:any;
+declare var sum:any;
+declare var toJSON:any;
 
 
 export function setUpDesignDocuments(database:any, callback:any) {
```

Comment:
Möchtest du wirklich 1x setUpDesignDocuments aufrufen und dann 6x die callback funktion?
Wenn in der callback weitere code ausgeführt wird.. wird dies 6x wiederholt..